packages/fluent-theme/src/components/activity/ActivityDecorator.module.css

```diff
@@ -0,0 +1,301 @@
+
+:global(.webchat-fluent) .activity-decorator {
```

Comment:
Should we nest everything under `:global(.webchat-fluent)` to simplify this CSS?

Reply:
We could, I just don't want to introduce the third level as native CSS nesting works quite different [compared to SASS](https://sass-lang.com/blog/sass-and-native-nesting/)